test/e2e/README.md

```diff
@@ -0,0 +1,50 @@
+# Pre-requisites
+
+The test doesn't yet manage the lifecycle of the cluster being tested,
+therefore the test hosts' kubeconfig must point to a Kubernetes cluster.
+You can verify that the kubeconfig is setup correctly simply with
+`kubectl get nodes`.
+
+The cluster under test must meet the following requirements:
+* Have 3 nodes
+* Each node must be configured per the quick start:
+  * At least 512 2MiB hugepages available
```

Comment:
There were some discussions regarding this. I'm thinking more generic and maybe more correct would be:
```suggestion
  * At least 1GiB of hugepages available (e.g. 512 2MiB hugepages)
```

Reply:
👍